src/Core/src/Handlers/Application/ApplicationHandler.iOS.cs

```diff
@@ -44,6 +44,21 @@ public partial class ApplicationHandler : ElementHandler<IApplication, IUIApplic
 			}
 		}
 
+		[SupportedOSPlatform("maccatalyst13.0")]
+		public static partial void MapActivateWindow(ApplicationHandler handler, IApplication application, object? args)
+		{
+			if (args is IWindow window)
+			{
+				var sceneSession = (window.Handler?.PlatformView as UIWindow)?.WindowScene?.Session;
+
+				if (sceneSession is not null)
+				{
+					UISceneSessionActivationRequest activationRequest = UISceneSessionActivationRequest.Create(sceneSession);
+					UIApplication.SharedApplication.ActivateSceneSession(activationRequest, errorHandler: null);
```

Comment:
I wonder if the error handler should be a basic logger message just so something happens.